Common/Orders/OrderEvent.cs

```diff
@@ -111,6 +121,25 @@ public OrderEvent(Order order, string message = "")
         }
 
         /// <summary>
+        /// Helper Constructor for TradeBuilder tests
+        /// </summary>
+        /// <param name="orderId">The Id of the order</param>
+        /// <param name="symbol">Asset Symbol</param>
+        /// <param name="time">Date/time of this event</param>
+        /// <param name="fillPrice">Fill price</param>
+        /// <param name="fillQuantity">Fill quantity</param>
+        /// <param name="orderFee">The order fee</param>
+        public OrderEvent(int orderId, string symbol, DateTime time, decimal fillPrice, int fillQuantity, decimal orderFee)
```

Comment:
The fields added to the OrderEvent class are actually two: UtcTime and OrderFee (both are required by the TradeBuilder). I agree this constructor is only used for testing purposes and is similar to another existing one (also used only in testing). Maybe I could add these two parameters to the existing and remove this one ? The other constructor OrderEvent(Order) could be left untouched (at least for now).